api/src/pcapi/routes/backoffice/accounts/serialization.py

```diff
@@ -41,6 +44,15 @@ def get_fraud_check_eligibility_type(
     return users_models.EligibilityType.AGE17_18
 
 
+def _format_amount_for_recredit_action(amount: decimal.Decimal, user: users_models.User) -> str:
+    formatted_amount = f"{finance_utils.format_currency_for_backoffice(amount)} €"
+
+    if user is not None and user.is_caledonian:
+        formatted_amount += f" ({finance_utils.format_currency_for_backoffice(amount, True)} CFP)"
```

Comment:
... ce qui donnerait : 
```suggestion
    formatted_amount = finance_utils.format_currency_for_backoffice(amount)

    if user is not None and user.is_caledonian:
        formatted_amount += f" ({finance_utils.format_currency_for_backoffice(amount, use_xpf=True)}"
```